protocol/avdl/chat1/local.avdl

```diff
@@ -94,20 +94,23 @@ protocol local {
   }
 
   record MessageInfoLocal {
-    boolean isNew;
```

Comment:
It was just removed. I noticed that the chat api wasn't using that field (plz correct me if wrong), and we are not populating that filed in every MessageInfoLocal, so it can be a bit confusing. It seemed to me that just using the reader info from conversation is easier. What do you think?